### PR TITLE
FAQ: merge both entries on account deletion

### PR DESCRIPTION
**Pull request by missytake**
follow-up on #953

### CODE REVIEW

**Review by hpk42 — APPROVED** [review]
very good improvement!

**Review by adbenitez — APPROVED** [review]
really nice

File: en/help.md [hunk 1]
```diff
@@ -1200,16 +1187,32 @@ Otherwise, deleting messages or multi-device setups might not work properly.
   [Autocrypt-enabled e-mail app](https://autocrypt.org/dev-status.html).
 
 
-### How can I delete my account?
+### How can I delete my account? {#remove-account}
+
+If you use a default chat profile
+you can simply uninstall the app.
+This will automatically trigger deletion of all associated account data on the chatmail server.
```
Comment (adbenitez) [inline en/help.md:1194]: this sounds like the deletion is immediate and not that there is a delay until the account is really deleted

File: en/help.md [hunk 2]
```diff
@@ -1200,16 +1187,32 @@ Otherwise, deleting messages or multi-device setups might not work properly.
...
+For more info, please refer to [nine.testrun.org account-deletion](https://nine.testrun.org/info.html#account-deletion) for the default onboarding server,
```
Comment (adbenitez) [inline en/help.md:1195]: is that link/change not deployed yet? clicked it but it doesn't show such section

**Comment by Simon-Laux** [comment]
Don't forget to also update the actual privacy policy (I doubt that google will accept FAQ, also for general legal safety it makes sense to mention privacy policy of chatmail servers in the privacy policy). My rough ideas from the internal chat:

> Like mention chatmail on [https://delta.chat/en/gdpr](https://github.com/deltachat/deltachat-pages/pull/954#): "email providers and chatmail providers have their own privacy policy."
and talk about "new profile creation creates accounts on the default instance nine.testrun.org, read it's privacy policy here: [https://nine.testrun.org/privacy.html](https://github.com/deltachat/deltachat-pages/pull/954#)"

File: en/help.md (hunk 2 above)
Comment (hpk42) [reply en/help.md:1195]: Right it's not.  We need to tedploy nnine first. But we can merge faq anyway.

File: en/help.md (hunk 1 above)
Comment (hpk42) [reply en/help.md:1194]: Not in my ears. The cloud often takes a while to really delete anything

**Comment by hpk42** [comment]
Let's still merge and consider privacy policy separately. Not sure yet if it's required.

**Pull request merged**